manifests/backend.pp

```diff
@@ -190,6 +190,8 @@
     configure                 => true,
     configure_url             => $sensu::api_url,
     configure_trusted_ca_file => $sensu::trusted_ca_file,
+    provider                  => 'sensu_api',
```

Comment:
Is it necessary to specify the provider? Isn't there only one?

Reply:
There are two providers for all types except those prefixed with `sensuctl`.  They are `sensuctl` and `sensu_api`.  By using the API to manage the admin user, removes potential circular dependency as sensuctl requires the admin user password and setting the admin user password via sensuctl would create a dependency problem when removing the `old_password` necessity. The API is able to either use existing `password` or reads tokens from the config files written by `sensuctl` which will exist after bootstrap and those are the "backup" form of API authentication on the backend host.

Reply:
Also the default provider for all types is `sensuctl` as a form of backwards compatibility before the API provider existed.  In order to use the API you must specify the `sensu_api` provider.